src/luascript.cpp

```diff
@@ -14519,25 +14519,30 @@ int LuaScriptInterface::luaMonsterTypeGetSummonList(lua_State* L)
 	int index = 0;
 	lua_createtable(L, monsterType->info.summons.size(), 0);
 	for (const auto& summonBlock : monsterType->info.summons) {
-		lua_createtable(L, 0, 3);
+		lua_createtable(L, 0, 6);
 		setField(L, "name", summonBlock.name);
 		setField(L, "speed", summonBlock.speed);
 		setField(L, "chance", summonBlock.chance);
+		setField(L, "effect", summonBlock.effect);
+		setField(L, "masterEffect", summonBlock.masterEffect);
+		setField(L, "max", summonBlock.max);
 		lua_rawseti(L, -2, ++index);
 	}
 	return 1;
 }
 
 int LuaScriptInterface::luaMonsterTypeAddSummon(lua_State* L)
 {
-	// monsterType:addSummon(name, interval, chance[, max = -1])
+	// monsterType:addSummon(name, interval, chance[, effect = CONST_ME_TELEPORT[, masterEffect = CONST_ME_NONE[, max = -1]]])
```

Comment:
Shouldn't the two new parameters go to the end, to maintain backwards compatibility?

Reply:
it should, good call